Ejercicio2/Gun.java

```diff
@@ -0,0 +1,23 @@
+package Ejercicio2;
+
+public class Gun extends Weapon{
+    public Ammo ammoType;
+
+    public Ammo getAmmoType() {
+        return ammoType;
+    }
+
+    public void setAmmoType(Ammo ammoType) {
+        this.ammoType = ammoType;
+    }
+
+    @Override
+    public String getType() {
+        return null;
+    }
+
+    @Override
+    public void setType(String type) {
+
```

Comment:
Yo need to set the attribute type. It is available because you are extending weapon